vernac/metasyntax.mli

```diff
@@ -24,6 +24,11 @@ val add_notation : locality_flag -> constr_expr ->
 
 val add_notation_extra_printing_rule : string -> string -> string -> unit
 
+(** Fine-grained activation and deactivation of notations *)
+
+val notation_activation :
```

Comment:
This API shouldn't be here but as it is a printing option it should go to constrintern.